client/src/pages/Contest/Contest.tsx

```diff
@@ -0,0 +1,204 @@
+  const { loggedInUser } = useAuth();
+
+  const newTheme = createMuiTheme({
+    palette: {
+      primary: {
+        main: '#000000',
+      },
+      secondary: {
+        main: '#3d3d3d',
+      },
+    },
+  });
+
+  interface TabPanelProps {
+    direction?: string;
+    value: number;
+    children: React.ReactNode;
+    index: number;
+  }
+
+  useEffect(() => {
+    async function getContestAllSubmissions() {
+      try {
+        const contestAllSubmissions = await getContestById(id);
+        if (contestAllSubmissions) {
+          setContestSubmissions(contestAllSubmissions);
+        }
+      } catch (err) {
+        console.log(err);
+      }
+    }
+    getContestAllSubmissions();
+  }, [id]);
+
+  const handleChange = (event: React.ChangeEvent<Record<string, unknown>>, valueChange: number) => {
+    setValue(valueChange);
+  };
+
+  const Panel = function (props: TabPanelProps) {
+    return (
+      <div role="tabpanel" id={`${props.index}`}>
+        {value === props.index && (
+          <Container>
+            <Typography>{props.children}</Typography>
+          </Container>
+        )}
+      </div>
+    );
+  };
+
+  const capitalize = (s: string) => s && s[0].toUpperCase() + s.slice(1);
+
+  return loggedInUser ? (
+    <>
+      <CssBaseline />
+      <GlobalCss />
+      <AuthHeader linkTo={`/submit-design/${id}`} btnText="SUBMIT DESIGN" />
+      <Container className={classes.container}>
+        <Grid className={classes.grid} container style={{ marginBottom: '35px' }}>
+          <Grid item>
+            <Typography>
+              {/* should be redirected to all the contests page*/}
+              <Link to="/discovery" className={classes.greyText}>
+                <ArrowBackIosIcon className={classes.backIcon} />
+                Back to contests list
+              </Link>
+            </Typography>
+          </Grid>
+        </Grid>
+        <Grid className={classes.grid} container>
+          <Grid item xs={10}>
+            <Typography className={classes.contestTitle}>
+              {contestSubmissions.title ? contestSubmissions.title : 'Lion tattoo concept in minimal style'}{' '}
+              <Button className={classes.prizeAmount}>
+                <Typography className={classes.prize}>
+                  {contestSubmissions.prizeAmount ? contestSubmissions.prizeAmount : '$150'}
+                </Typography>
+              </Button>
+            </Typography>
+            <Grid direction="row" className={classes.grid} container>
+              <Grid item>
+                <Avatar alt="Profile Image" src={ProfilePic} className={classes.avatar}></Avatar>
+              </Grid>
+              <Grid item>
+                <Typography className={classes.user}>By {capitalize(loggedInUser.username)}</Typography>
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid item xs={2}>
+            <Link to={`/submit-design/${id}`} style={{ textDecoration: 'none' }}>
```

Comment:
Link and Button can be combined here

Ref: https://stackoverflow.com/questions/51642532/how-to-make-a-material-ui-react-button-act-as-a-react-router-dom-link

Reply:
This is fixed.